iis-wf/iis-wf-export-actionmanager/src/main/java/eu/dnetlib/iis/wf/export/actionmanager/entity/facade/MDStoreFacade.java

```diff
@@ -0,0 +1,23 @@
+package eu.dnetlib.iis.wf.export.actionmanager.entity.facade;
+
+import eu.dnetlib.data.mdstore.DocumentNotFoundException;
+import eu.dnetlib.data.mdstore.MDStoreServiceException;
+
+/**
+ * MDStore service facade.
```

Comment:
Again, some more information would be helpful. For example, when one should write an implementation of this class.